Makefile

```diff
@@ -1,9 +1,10 @@
-dev:
+dev-build:
 	docker build --rm -f Dockerfile.dev -t jb_hugo:latest .
```

Comment:
pulling out the build definition from each of the subsequent dev definitions allows you to re-use this step in other places.